src/_pytest/fixtures.py

```diff
@@ -1074,6 +1073,24 @@ def __repr__(self) -> str:
         )
 
 
+class IdentityFixture(FixtureDef[FixtureValue]):
+    def __init__(
+        self,
+        fixturemanager: "FixtureManager",
+        argname: str,
+        scope: Union[Scope, _ScopeName, Callable[[str, Config], _ScopeName], None],
```

Comment:
_ispytest should be a false default passtrugh to ensure callers inside of pytest can set it and outside callers wont get to call it without a private arg